database/languages/c++.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": "c++ Tutorial",
+        "description": " Learn the basics of C++ it is a popular programming language.C++ is used to create computer programs, and is one of the most used language in game development.",
```

Comment:
[Recommendation] This description mostly describes what C++ is. While helpful, I highly suggest describing **why** people in tech should use this source. 
```suggestion
        "description": " Learn the basics of C++ it is a popular programming language.C++ is used to create computer programs, and is one of the most used languages in game development.",
```